### DISCUSSION TITLE
Вопросы по Lab_Task результата, Lab_Observation, Lab_Composition

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#260] [Лабораторная РГ]
В предложениях лабораторной группы значится, что output.code используется в профиле:
<img width="523" alt="image" src="https://github.com/fhir-ru/core/assets/62525329/2c882fe3-0665-4d22-8145-d44ef549152c">
и предложено сделать справочник из одного значения: DR - diagnostic report.

Видимо, надо привести пример, тогда будет более понятен смысл этого предложения.

**Comment by eugenekogan-spb** [comment]
2. Аналогичный вопрос о поле performerType. coding.system
<img width="801" alt="image" src="https://github.com/fhir-ru/core/assets/62525329/6e065b77-260d-4d9b-ac04-4ef501fb4ed1">

Предлагается вместо Snomed взять справочник МЗ РФ 
1.2.643.5.1.13.13.11.1034 - Виды медицинской помощи

**Comment by GorbachevaT** [comment]
[1.2.643.5.1.13.13.11.1034_4.2.xlsx](https://github.com/fhir-ru/core/files/14731304/1.2.643.5.1.13.13.11.1034_4.2.xlsx)
Вот содержимое справочника Виды медицинской помощи

**Reply by eugenekogan-spb** [reply]
<img width="410" alt="image" src="https://github.com/fhir-ru/core/assets/62525329/4a0b4989-3db6-4d2c-ab30-c7ac383fcdab">

**Reply by eugenekogan-spb** [reply]
Выглядит как не совсем справочник типов исполнителей.

**Comment by eugenekogan-spb** [comment]
3. Из профиля Lab_Observation, кажется, следует убрать referenceRange.high/low.comparator, потому что он недопустим для simpleQuantity

<img width="620" alt="image" src="https://github.com/fhir-ru/core/assets/62525329/9bbfe3d5-4483-4ce6-bea4-ea7cf571191a">

**Comment by eugenekogan-spb** [comment]
4. В Lab_Composition.confidentiality предлагается использовать справочник МЗ РФ
<img width="666" alt="image" src="https://github.com/fhir-ru/core/assets/62525329/3437d073-5395-4e92-8c77-95d6146a7232">

Это сделать не удается, потому что тип данных code.
